src/Aspire.Hosting/Aspire.Hosting.csproj

```diff
@@ -37,6 +37,7 @@
     <Compile Include="$(SharedDir)LaunchProfile.cs" Link="LaunchProfile.cs" />
     <Compile Include="$(SharedDir)LaunchSettingsSerializerContext.cs" Link="LaunchSettingsSerializerContext.cs" />
     <Compile Include="$(SharedDir)SecretsStore.cs" Link="Utils\SecretsStore.cs" />
+    <Compile Include="$(SharedDir)ConsoleLogs\*.cs" LinkBase="Utils\ConsoleLogs" />
```

Comment:
Adding with a wildcard like this could, in future, bring in bits we don't intend to be used nor need. Consider being explicit.

Reply:
```suggestion
    <Compile Include="$(SharedDir)ConsoleLogs\TimestampParser.cs" Link="Utils\ConsoleLogs\TimestampParser.cs" />
```